src/traces/ohlc/attributes.js

```diff
@@ -0,0 +1,105 @@
+/**
+* Copyright 2012-2016, Plotly, Inc.
+* All rights reserved.
+*
+* This source code is licensed under the MIT license found in the
+* LICENSE file in the root directory of this source tree.
+*/
+
+
+'use strict';
+
+var Lib = require('../../lib');
+var scatterAttrs = require('../scatter/attributes');
+
+var lineAttrs = scatterAttrs.line;
+
+var directionAttrs = {
+    visible: {
+        valType: 'enumerated',
+        values: [true, false, 'legendonly'],
+        role: 'info',
+        dflt: true,
+        description: [
+
+        ].join(' ')
+    },
+
+    color: Lib.extendFlat({}, lineAttrs.color),
+    width: Lib.extendFlat({}, lineAttrs.width),
+    dash: Lib.extendFlat({}, lineAttrs.dash),
+
+    tickwidth: {
+        valType: 'number',
+        min: 0,
+        max: 1,
```

Comment:
They'll start to overlap at 0.5, right? Not sure if we want to *forbid* overlap, but normally you wouldn't want to go past this.